Rules.md

```diff
@@ -203,15 +203,16 @@ A battle has one or more rounds. Every round has following parts:
 1. **Attacker decides to retreat or has no healthy units left:** All units move back to the last
    touched unoccupied, sea or owned region.
 1. **Defender decides to retreat or has no healthy units left:** All units (including wounded units)
-   move to an adjacent land region that is unoccupied or owned by the defender.
+   move to a single adjacent land region that is unoccupied or owned by the defender. Exception:
+   units can't retreat to the region where the attacker's move token was resolved.
```

Comment:
I think my comments were for this thread.

If you take the example of Russia, Ukraine and Poland I described earlier:

> You can hop from Russia to Ukraine to attack Poland. Can the defender retreat to Russia? --> The current rules make this clear, your suggestion leaves it a bit ambiguous.

If we evaluate the proposed rules:
> "units can't retreat to the region where the attacker's move token was resolved"
The token was resolved in Russia and so the defender can't retreat to Russia
> "units can't retreat to the region where the attacker came from"
Since "came from" isn't defined rigorously, this could mean Ukraine and/or Russia. --> Ambiguous